server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1723,7 +1728,7 @@ static String createEmptyTranslog(Path location, long initialGlobalCheckpoint, S
         final String translogUUID = UUIDs.randomBase64UUID();
         TranslogWriter writer = TranslogWriter.create(shardId, translogUUID, 1, location.resolve(getFilename(1)), channelFactory,
             new ByteSizeValue(10), 1, initialGlobalCheckpoint,
-            () -> { throw new UnsupportedOperationException(); }, () -> { throw new UnsupportedOperationException(); }
+            () -> { throw new UnsupportedOperationException(); }, () -> { throw new UnsupportedOperationException(); }, 0L
```

Comment:
shall we make people pass the term to this method? If it's difficult, it's not a big deal but I feel it will be good to have the right term on the file.

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/29227/commits/ad5a2123a37c4400b1d607669078d9cac09a32d3